hummingbot/connector/exchange/okx/okx_auth.py

```diff
@@ -61,7 +61,9 @@ def _generate_signature(self, timestamp: str, method: str, path_url: str, body:
         return signature
 
     def authentication_headers(self, request: RESTRequest) -> Dict[str, Any]:
-        timestamp = datetime.utcfromtimestamp(self.time_provider.time()).isoformat(timespec="milliseconds") + "Z"
+        timestamp = datetime.utcfromtimestamp(self.time_provider.time()).isoformat(timespec="milliseconds")
```

Comment:
why we have this change? is tested?

Reply:
Ok, this did fail the tests. The `Z` conflicts with the timezone aware `isoformat()`, i.e. `YYYY-MM-HH:MM:SS+00:00`. This cannot have a `Z` at the end. It may not come from `cython` per-se, but associated updates in the .yml

Ok, I know I may not be the that good dev, but comeon ... of course it is tested. The way I test: I uninstall `hummingbot` env, install it from scratch, then clean/compile/make test ... plus does it not pass the workflow!